file_paths.md

```diff
@@ -163,160 +163,201 @@ The package will create the following services `puppet`, `mcollective`,
+            curl.exe *
+            curl-config *
+            daemon.bat *
+            daemon.rb *                           # windows service daemon
+            elevate.exe *                         # Used to elevate interactive commands
+            elevate.exe.config *
+            environment.bat *                     # setup LOAD_PATH, globally used by all our .bat files, to be symlinked to C:\Program Files\Puppet Labs\bin
+            erb.bat *
+            extlookup2hiera *
+            extlookup2hiera.bat *
+            facter.bat *                          # bat file wrapper
             facter.exe *
+            facter_interactive.bat *
+            gem.bat *
+            hiera *                               # ruby bin wrapper
+            hiera.bat *                           # bat file wrapper
+            irb.bat *
+            libcpp-pcp-client.so *
+            libcurl-4.dll *                       # curl dll
+            libeay32.dll *                        # openssl dll
             libfacter.so *
-            lib*.dll *                        # gcc, mingw, boost libraries
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
+            libgcc_s_seh-1.dll *
+            libstdc++-6.dll *
+            libwinpthread-1.dll *
+            mco *                                 # ruby bin wrapper
+            mco.bat *                             # bat file wrapper
+            mco_daemon.bat *
+            mcollectived *                        # ruby bin wrapper
+            minitar *
+            minitar.bat *
+            nssm.exe *                            # NSSM used to run pxp-agent
+            openssl.exe *
+            puppet *                              # ruby bin wrapper
+            puppet.bat *                          # bat file wrapper
+            puppet_interactive.bat *
+            puppet_shell.bat *                    # targets for shortcuts
+            puppetres.dll *                       # event log message resource dll
+            pxp-agent.exe *
+            rake.bat *
+            rdoc.bat *
+            ri.bat *
+            ruby.exe *
+            rubyw.exe *
+            run_facter_interactive.bat *
+            run_puppet_interactive.bat *
+            ssleay32.dll *                        # openssl dll
+            stompcat *
+            stompcat.bat *
+            testrb.bat *
+            x64-msvcrt-ruby210.dll *              # ruby dll
+            zlib1.dll *
 
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
+        include *
+            boost *
```

Comment:
The note on boost libraries above looks like a generalization of finding libboost_nowide.dll, which we stopped shipping. A recent MSI doesn't contain any Boost binaries, as we statically link them rather than shipping DLLs.

```
C:\Program Files\Puppet Labs\Puppet> ls .\facter\bin\


    Directory: C:\Program Files\Puppet Labs\Puppet\facter\bin


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        1/22/2016   2:24 PM         747851 facter.exe
-a----        1/22/2016   2:24 PM        5044268 libfacter.so
-a----        1/22/2016   2:24 PM        7950655 libfacter_test.exe
-a----        1/22/2016   2:24 PM          80384 libgcc_s_seh-1.dll
-a----        1/22/2016   2:24 PM         964096 libstdc++-6.dll
-a----        1/22/2016   2:24 PM          42675 libtest.so
-a----        1/22/2016   2:24 PM          43784 libtest1.so
-a----        1/22/2016   2:24 PM          54272 libwinpthread-1.dll
-a----        1/22/2016   2:24 PM          89277 lth_cat.exe
-a----        1/22/2016   2:24 PM             18 VERSION


C:\Program Files\Puppet Labs\Puppet> ls .\pxp-agent\bin\


    Directory: C:\Program Files\Puppet Labs\Puppet\pxp-agent\bin


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        1/22/2016   2:24 PM        4472841 cpp-pcp-client-unittests.exe
-a----        1/22/2016   2:24 PM       16233621 libcpp-pcp-client.so
-a----        1/22/2016   2:24 PM          80384 libgcc_s_seh-1.dll
-a----        1/22/2016   2:24 PM          72401 libnowide.dll
-a----        1/22/2016   2:24 PM         964096 libstdc++-6.dll
-a----        1/22/2016   2:24 PM          54272 libwinpthread-1.dll
-a----        1/22/2016   2:24 PM          63295 lth_cat.exe
-a----        1/22/2016   2:24 PM        4133441 pxp-agent-unittests.exe
-a----        1/22/2016   2:24 PM        3285563 pxp-agent.exe
-a----        1/22/2016   2:24 PM             18 VERSION
```

We should not be including Boost include files or binaries in the MSI.